ddmrp/models/stock_warehouse_orderpoint.py

```diff
@@ -315,6 +337,19 @@ def _compute_dlt(self):
 
     _order = 'planning_priority_level asc, net_flow_position asc'
 
+    @api.model
+    def create(self, vals):
+        record = super(StockWarehouseOrderpoint, self).create(vals)
+        record._calc_adu()
+        return record
+
+    @api.multi
+    def write(self, vals):
+        super(StockWarehouseOrderpoint, self).write(vals)
+        if not self.env.context.get('__calc_adu'):
```

Comment:
can we make this ctx key more meaningful? I don't expect `__calc_adu` to disable the calc :)